### DISCUSSION TITLE
Page Conversion resulting in zero structure rules

### DISCUSSION THREAD

**Discussion by subho4523** [adobe/aem-modernize-tools#185] [Q&A]
Hello,

I am using Modernizer 2.1.1.SNAPSHOT in my cloud instance. I have created a basic template using geodemo provided by Adobe. I am currently trying to move my /apps/geometrixx/templates/homepage (static) to /conf/geodemo/settings/wcm/templates/geodemo-home-page(editable). I used layout containers  to recreate the geometrix page. How ever after i choose configuration as the conf/geodemo my page shows zero conversions. I get no logs. Below is the screenshot. Any help Thanks in Advance.

Regards
jayanta

![image](https://user-images.githubusercontent.com/105743027/168887281-d0cd42df-962d-446d-bf81-b81e331ae59e.png)

**Comment by bstopp** [comment]
Did you create an [OSGI Configuration](https://opensource.adobe.com/aem-modernize-tools/pages/structure/config.html) to map the Static Template to the Editable Template?
